### PR TITLE
palindrome string check assignment

### PR DESCRIPTION
**Pull request by HiralBhalani**

### CODE REVIEW

File: hiral's work/Assisgnments/palindromstring.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+# function to replace the space and test for palindrome string
+def palindrome_string_check(user_str):
+    user_str = user_str.replace(" ", "")
```
Comment (Patibandha) [inline hiral's work/Assisgnments/palindromstring.py:4]: this is good, but what about the special characters?

File: hiral's work/Assisgnments/palindromstring.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    print(user_str)
+    return user_str == user_str[::-1]
+
+
+# getting user string input
+s = input("Please enter your string: ")
+tested_s = palindrome_string_check(s)
```
Comment (Patibandha) [inline hiral's work/Assisgnments/palindromstring.py:11]: follow batter naming convention

**Comment by Patibandha** [comment]
also resolve merge conflict before merge!

File: hiral's work/Assisgnments/palindromstring.py (hunk 1 above)
Comment (Patibandha) [reply hiral's work/Assisgnments/palindromstring.py:4]: no update!!!!!

**Review by Patibandha — COMMENTED** [review]
test cases 5 per assignment

File: hiral's work/Assisgnments/palindromstring.py (hunk 1 above)
Comment (HiralBhalani) [reply hiral's work/Assisgnments/palindromstring.py:4]: done

File: hiral's work/Assisgnments/palindromstring.py (hunk 2 above)
Comment (HiralBhalani) [reply hiral's work/Assisgnments/palindromstring.py:11]: done.

**Pull request closed without merging**